master/internal/db/postgres_experiments_intg_test.go

```diff
@@ -585,10 +585,10 @@ func TestTopTrialsByMetric(t *testing.T) {
 		`[{"a":-1.5, "b":1.0, "c":"Infinity"}]`, false)
 
 	const (
-		more             = false
-		less             = true
-		noError          = true
-		error            = false
+		moreExpected     = false
+		lessExpected     = true
```

Comment:
```suggestion
		lessIsBetter     = true
		moreIsBetter     = false
```
I found these new names to be *more* confusing, since they're not used as an expected value in a test assertion or a conditional related to one like `errorExpected` and `noErrorExpected` are.